pkg/3scale/amp/component/system_options.go

```diff
@@ -93,6 +93,7 @@ type SystemOptions struct {
 	MemcachedLabels          map[string]string `validate:"required"`
 	SMTPLabels               map[string]string `validate:"required"`
 	SideKiqMetrics           bool
+	AppMetrics               bool
```

Comment:
Created AppMetrics option to be able to enable this only on operator side and not on templates side, following same approach as SidekiqMetrics